openpype/hosts/blender/plugins/load/load_abc.py

```diff
@@ -60,18 +60,35 @@ def _process(self, libpath, asset_group, group_name):
 
         imported = lib.get_selection()
 
-        # Children must be linked before parents,
-        # otherwise the hierarchy will break
+        empties = [obj for obj in imported if obj.type == 'EMPTY']
+
+        container = None
+
+        for empty in empties:
+            if not empty.parent:
+                container = empty
+                break
```

Comment:
Would this be the same but faster?
```suggestion
        container = next(
            (obj for obj if obj.type == "EMPTY" and not obj.parent),
            None
        )
```